components/financial_assistance/app/views/financial_assistance/applicants/_dependent_form.html.erb

```diff
@@ -14,6 +14,7 @@
               <ul>
                 <% @applicant.errors.full_messages.each do |msg| %>
                     <% msg = (msg == "Gender can't be blank" && EnrollRegistry.feature_enabled?(:gender_to_sex)) ? "Sex can't be blank" : msg %>
+                    <% msg = (msg == "ssn is missing") ? "SSN is missing" : msg %>
```

Comment:
are we missing some translations here as well?

Reply:
The fix for gender can't be blank didn't end up being in the same file(s) as ssn is missing, so I'm gonna leave that change for another ticket/PR